src/index.ts

```diff
@@ -141,6 +141,22 @@ export const iconsSpritesheet: (args: PluginProps) => Plugin = ({ withTypes, inp
   apply(config) {
     return config.mode === "development";
   },
+  async configResolved(config) {
+    const outputSvgPath = normalizePath(path.join(cwd ?? process.cwd(), outputDir, fileName ?? "sprite.svg"));
+    if (
+      !(await fs
+        .access(outputSvgPath, fs.constants.F_OK)
+        .then(() => true)
+        .catch(() => false))
```

Comment:
could we extract this into a const right above this? I don't really like to have it in the if statement?

Reply:
Done, extracted into a const. Thanks

Reply:
Awesome work. Thank you on this!